Modules/System/Azure Blob Services API/src/ABSContainerClient.Codeunit.al

```diff
@@ -120,6 +120,201 @@ codeunit 9052 "ABS Container Client"
         exit(BlobServicesApiImpl.DeleteContainer(ContainerName, OptionalParameters));
     end;
 
+    /// <summary>
+    /// Requests a new lease. If the container does not have an active lease, the blob service creates a lease on the container. The lease duration can be 15 to 60 seconds or can be infinite
+    /// see: https://docs.microsoft.com/en-us/rest/api/storageservices/lease-container
+    /// </summary>
+    /// <returns>An operation reponse object</returns>
+    procedure LeaseAcquire(ContainerName: Text): Codeunit "ABS Operation Response"
+    var
+        OptionalParameters: Codeunit "ABS Optional Parameters";
+        ProposedLeaseId: Guid;
+    begin
+        exit(LeaseAcquire(ContainerName, -1, ProposedLeaseId, OptionalParameters)); // Infinite duration, null Guid
+    end;
+
+    /// <summary>
+    /// Requests a new lease. If the container does not have an active lease, the blob service creates a lease on the container. The lease duration can be 15 to 60 seconds or can be infinite
+    /// see: https://docs.microsoft.com/en-us/rest/api/storageservices/lease-container
+    /// </summary>
+    /// <param name="OptionalParameters">Optional parameters to pass.</param>
+    /// <returns>An operation reponse object</returns>
+    procedure LeaseAcquire(ContainerName: Text; OptionalParameters: Codeunit "ABS Optional Parameters"): Codeunit "ABS Operation Response"
+    var
+        ProposedLeaseId: Guid;
+    begin
+        exit(LeaseAcquire(ContainerName, -1, ProposedLeaseId, OptionalParameters)); // Infinite duration, null Guid
+    end;
+
+    /// <summary>
+    /// Requests a new lease. If the container does not have an active lease, the blob service creates a lease on the container. The lease duration can be 15 to 60 seconds or can be infinite
+    /// see: https://docs.microsoft.com/en-us/rest/api/storageservices/lease-container
+    /// </summary>    
+    /// <param name="DurationSeconds">Specifies the duration of the lease, in seconds, or negative one (-1) for a lease that never expires</param>
+    /// <returns>An operation reponse object</returns>
+    procedure LeaseAcquire(ContainerName: Text; DurationSeconds: Integer): Codeunit "ABS Operation Response"
+    var
+        OptionalParameters: Codeunit "ABS Optional Parameters";
+        ProposedLeaseId: Guid;
+    begin
+        exit(LeaseAcquire(ContainerName, DurationSeconds, ProposedLeaseId, OptionalParameters)); // Custom duration, null Guid
```

Comment:
Why note call the implementation here directly? To save a couple of function calls.

Reply:
Changed it